src/test/eo/org/eolang/math/number-tests.eo

```diff
@@ -648,6 +648,65 @@
     .power 10
     $.equal-to 2655.992279142402
 
+[] > sqrt-check-1
+  assert-that > @
+    abs.
+      number
+        minus.
+          0.0
+          sqrt.
+            number 0.0
+    $.less-than 0.00000000001
+
+[] > sqrt-check-2
+  assert-that > @
+    is-nan.
+      sqrt.
+        number -0.1
+    $.equal-to TRUE
+
+[] > sqrt-check-3
+  assert-that > @
+    abs.
+      number
+        minus.
+          2.0
+          sqrt.
+            number
+              4
+    $.less-than 0.00000000001
+
+[] > sqrt-check-4
+  assert-that > @
+    abs.
+      number
+        minus.
+          10000.0
+          sqrt.
+            number
+              100000000
+    $.less-than 0.00000000001
```

Comment:
@levBagryansky are `sqrt-check-4` and `sqrt-check-3` not the same tests?

Reply:
@includealex The test checks sqrt of  big number

Reply:
@levBagryansky so why we are not having test with max integer?

Reply:
@includealex We do not test such inputs for any function (neither `pow`, nor `signum`, nor `arccos`). Looks like this test doesn't make sense. Sure I can add it if you believe it is necessary.

Reply:
@levBagryansky if you want to test `sqrt` on big integers, won't changing big number in this test to max integer be great idea?

Reply:
@includealex First of all, I want to check exactly `int`. So if we want to check specifically big integer, I guess would be better to create a separate test about it. Decision to add new test is up to you

Reply:
@levBagryansky as you said, `sqrt-check-4` test was made for testing big integers. Maybe it would be better to have bigger integers testing?

Reply:
@includealex please check my previous comment. I have already said it is better to create a separate test to check big integer. `sqrt-check-4` checks `int` input and big integer plays no role